Podfile

```diff
@@ -8,14 +8,25 @@ target 'WatchFaceDumper' do
 end
 
 def lift_to_xcode_recommended_settings(pi)
-  pi.pods_project.targets.each do |t|
-    t.build_configurations.each do |c|
-      c.build_settings['MACOSX_DEPLOYMENT_TARGET'] = '10.15'
+  # to identify whether it is Xcode 15
+  xcode_base_version = `xcodebuild -version | grep 'Xcode' | awk '{print $2}' | cut -d . -f 1`
+
+  pi.pods_project.targets.each do |target|
+    target.build_configurations.each do |config|
+      config.build_settings['MACOSX_DEPLOYMENT_TARGET'] = '10.15'
       # suppress recommended settings warnings: automatically select archs
-      c.build_settings.delete 'ARCHS'
+      config.build_settings.delete 'ARCHS'
+      
+      # for xcode 15+ only
+      if config.base_configuration_reference && Integer(xcode_base_version) >= 15
+        xcconfig_path = config.base_configuration_reference.real_path
+        xcconfig = File.read(xcconfig_path)
+        xcconfig_mod = xcconfig.gsub(/DT_TOOLCHAIN_DIR/, "TOOLCHAIN_DIR")
```

Comment:
this workaround should not be needed as CocoaPods 1.13.0 handles the change.
https://github.com/CocoaPods/CocoaPods/releases/tag/1.13.0

could you try with CocoaPods 1.13.0+ removing the workaround?

Reply:
Have tried on higher version on separate machine still needed.

Reply:
I have tried on a higher version on a separate machine, but it still needed.

Reply:
I have tried on a higher version on a separate machine, but it is still needed.